ui/src/components/layout/layout.module.css

```diff
@@ -1,5 +1,5 @@
 .layout {
-	@apply w-full h-full grid;
+	composes: w-full h-full grid from global;
```

Comment:
`@apply` copies the Tailwind styles into the class, while `composes` is a CSS Module feature that makes those classes part of the CSS module. This change makes style repeating less frequent by reusing TailwindCSS's classes.